src/main/webapp/feed.js

```diff
@@ -166,24 +166,71 @@ function addEventBookmark(event) {
     bookmarkDiv.style.height = 0;
     const bookmark = document.createElement('img');
     bookmark.className = "event-bookmark";
-    // add case for if user has bookmarked this event once users have been created
-    bookmark.src = "/resources/bookmark.png";
     bookmarkDiv.appendChild(bookmark);
     bookmarkDiv.appendChild(addEventNumBookmarks(event));
+    bookmarkDiv.onclick = () => {};
+
+    if (gapi.auth2.getAuthInstance().isSignedIn.get() && user.bookmarked_events.includes(event.extendedProperties.event_id)) {
+        bookmark.src="/resources/filled-bookmark.png"; 
+        bookmarkDiv.childNodes[1].style.color = "#fafafa";
```

Comment:
Why in all these cases is it childNodes[1], could there be a constant set to 1 that would make this more clear?